src/helix-ui/icons/server-incident.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
🎨 Previous icon `technical-incident` had remnant design of the dark days. Bringing up to speed with modern times.